pandora/entity_ids.gd

```diff
@@ -0,0 +1,3 @@
+# Do not modify! Auto-generated file.
+class_name EntityIds
```

Comment:
Fix this: It should not generate this unless generate ID class is explicitly toggled